src/catchup/VerifyLedgerChainWork.h

```diff
@@ -24,28 +24,36 @@ class VerifyLedgerChainWork : public Work
     TmpDir const& mDownloadDir;
     LedgerRange mRange;
     uint32_t mCurrCheckpoint;
-    bool mManualCatchup;
     LedgerHeaderHistoryEntry& mFirstVerified;
-    LedgerHeaderHistoryEntry& mLastVerified;
+    LedgerHeaderHistoryEntry const& mLastClosedLedger;
+    optional<Hash> mTrustedHash;
+
+    // First ledger of last verified checkpoint. Needed for a checkpoint that
+    // is being verified: last ledger in current checkpoint must agree with
+    // mVerifiedAhead
+    LedgerHeaderHistoryEntry mVerifiedAhead{};
 
     medida::Meter& mVerifyLedgerSuccessOld;
     medida::Meter& mVerifyLedgerSuccess;
     medida::Meter& mVerifyLedgerFailureLedgerVersion;
     medida::Meter& mVerifyLedgerFailureOvershot;
+    medida::Meter& mVerifyLedgerFailureUndershot;
```

Comment:
I don't think we care to have that kind of granularity for metrics

Reply:
Updated this, and kept "ledger-verify" success and failure metrics only.